android/app/src/main/java/au/com/robin/sms/MainActivity.kt

```diff
@@ -8,6 +8,7 @@ import android.widget.EditText
 import android.widget.Toast
 import androidx.annotation.RequiresPermission
 import androidx.appcompat.app.AppCompatActivity
+import au.com.robin.sms.develop.R
```

Comment:
It's unfortunate that R needs to be based on PACKAGE_NAME

Reply:
Also, looking at `R.layout.activity_main`, where does `activity_main` come from?

Reply:
> It's unfortunate that R needs to be based on PACKAGE_NAME

Yes, I deliberately set the `namespace` to PACKAGE_NAME. As for `R.layout.activity_main`, it gets a unique identifier when android build system generates the `R` class. Since, I have `namespace` set to PACKAGE_NAME, we have to import that to refer to that R class. 

It is actually a bit confusing as we have different package's name, but I am doing that because I just want to try it out if it's going to build successfully and run without any problem on the real device. We can always change it back to what we need accordingly. 

I hope that answers your question!